numpy/lib/_arraysetops_impl.py

```diff
@@ -443,13 +438,10 @@ def unique_all(x):
 
     Examples
     --------
-    >>> import numpy as np
```

Comment:
The `>>> import numpy as np` statements should not be removed. These were recently added to all the docstrings and should remain there (see e.g. #26814).